[14.0][MIG] sale_procurement_amendment (#1574)
OCA/sale-workflow: sale_procurement_amendment/README.rst

    :alt: Production/Stable
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
    :target: https://github.com/OCA/sale-workflow/tree/14.0/sale_procurement_amendment
    :alt: OCA/sale-workflow
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_procurement_amendment
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/167/14.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

In Odoo, when you confirm a Sale Order, you can increase the quantities
in sale lines. This is reflected in Pickings.
But, if you decrease the quantities, you have to do this manually even if
nothing has been done in any picking.

This allows to reflect confirmed sale lines quantities

> amendments to procurements (and pickings) if nothing has been already begun

Reviewer: ```suggestion
amendments to procurements (and pickings) if nothing was already started
```
